### DISCUSSION TITLE
blog/2022-01-21_proxmox_root_migration/

### DISCUSSION THREAD

**Discussion by giscus** [Sieboldianus/du.nkel.dev#14] [Announcements]
# blog/2022-01-21_proxmox_root_migration/

A personal code notes blog

https://du.nkel.dev/blog/2022-01-21_proxmox_root_migration/

**Comment by Sieboldianus** [comment]
Comments welcome. If you do not want to allow Github API access for [giscus.app](https://giscus.app/), feel free to [directly add comments on Github](https://github.com/Sieboldianus/du.nkel.dev/discussions/14).
